src/mainwindow.cpp

```diff
@@ -234,6 +236,30 @@ void MainWindow::setup_ActionsMenu_Actions()
     setup_Action(MOVE_RIGHT, new QAction(tr("Move Tab &Right"), settingOwner),
                  MOVE_RIGHT_SHORTCUT, consoleTabulator, SLOT(moveRight()), menu_Actions, data);
 
+    menu_Actions->addSeparator();
+    data.setValue(checkHasIndexedTab);
+
+    setup_Action(TAB_1, new QAction(QIcon::fromTheme(QStringLiteral("go-1")), tr("Tab &1"), settingOwner),
```

Comment:
Some comments:

* Is it possible to use a loop instead of 10 function calls?
* Is there an example icon theme provides `go-*` icons?

Reply:
Yeah. I switched it to using QSignalMapper instead. This is indeed much nicer :-)

Not entirely sure about the icons. It seems a little weird to have an icon for a number. That kinda is your font, no? :-)

Reply:
Got it! Just FYI - QAction can be constructed without an icon ;-)

Reply:
Indeed. I removed the icon reference.